bin/process_vcf.py

```diff
@@ -115,7 +117,7 @@ def handle_indel(vcf_header, record):
     # apply the most frequent ALT. This is because there is evidence for /an/ indel but it is
     # ambiguous which one. We can't represent ambiguous indels in a consensus fasta so this
     # is the best we can do.
-    if sum(vafs) < 0.5:
+    if sum(vafs) < 0.6:
```

Comment:
Why was this changed from `0.5` to `0.6`?

Reply:
With testing more data a lot of the indels that were around 0.5 -  0.55 were low quality and caused frameshifts / broke the rule of 6. So adjusting the threshold to 0.6 to help create better sequences

It probably should be a parameter though so I'll have to add it

Reply:
That makes sense. It can be moved into a parameter later on.